css/spacing.css

```diff
@@ -2,10 +2,22 @@
     margin-top: 3.8rem;
 
 }
+.m-2t{
```

Comment:
Class names are not consistent.
- `.m-t` doesn't tell that it will give `3.8rem` margin
- similarly `.m-2t` doesn't tell it will give `2.5rem` margin
- similarly `.m-t5` is not consistent in naming.

class names should be more generic that can easily convey their meaning by class names.

Reply:
yeah you are right will try to change it

Reply:
hey i have changed it now ,  m-t3 for top 3.8rem , m-2t for top 2.5 and m-l0 for margin left 0 , thnku for pointing it out